shell/platform/windows/flutter_windows_view.h

```diff
@@ -200,10 +201,18 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate,
   // |WindowBindingHandlerDelegate|
   virtual ui::AXFragmentRootDelegateWin* GetAxFragmentRootDelegate() override;
 
+  // Called to re/set the accessibility bridge pointer.
+  virtual void UpdateSemanticsEnabled(bool enabled);
```

Comment:
The content of this new method could have instead been placed in `OnUpdateSemanticsEnabled`, which is what calls `FlutterWindowsEngine::UpdateSemanticsEnabled`. However, once we have multiple views, a call to `OnUpdateSemanticsEnabled` on a view would not necessarily call it for other views.